swap_meet/vendor.py

```diff
@@ -0,0 +1,95 @@
+from swap_meet.item import Item
+
+class Vendor: 
+    def __init__(self, inventory=None):
+        if inventory == None: 
+            self.inventory = []
+        else: 
+            self.inventory = inventory
+
+    def add(self, add_item):
+        self.inventory.append(add_item)
+        return add_item    
+    
+    def remove(self, rm_item): 
+        if rm_item in self.inventory:
+            self.inventory.remove(rm_item)
+            return rm_item
+        else:
+            return rm_item  == False 
+    
+    def get_by_category(self, asked_category):
+        item_list =[]
+        for item in self.inventory: 
+            if item.category == asked_category:
+                item_list.append(item)
+        
+        return item_list
+    
+    def swap_items(self, other_vendor, my_item, their_item): 
+        if my_item in self.inventory and their_item in other_vendor.inventory: 
+            self.remove(my_item)
+            other_vendor.add(my_item)
+            other_vendor.remove(their_item)
+            self.add(their_item)
+            return True 
+        else: 
+            return False
```

Comment:
The `else` isn't needed.

```suggestion
        return False
```